vita3k/main.cpp

```diff
@@ -195,6 +195,11 @@ int main(int argc, char *argv[]) {
     if (emuenv.cfg.controller_binds.empty() || (emuenv.cfg.controller_binds.size() != 15))
         gui::reset_controller_binding(emuenv);
 
+    if (!emuenv.cfg.controller_led_value.empty() && (emuenv.cfg.controller_led_value.size() != 3)) {
```

Comment:
Why 2 checks?

Reply:
Because for no get crash if is no empty and have only 2 value exemple

Reply:
Check here, is safety check, like for control binding, is for no get any crash caused by config edited or corrupted, if have exemple only one or 2 value

Reply:
i have removed this check for using other way for this case